### PR TITLE
Added Python 3.7 to the gitlab.yaml build file

### PR DESCRIPTION
**Pull request by abhishekpratapa**
## Overview
- Added Build Python 3.7 for Linux
- Added Test Python 3.7 for Linux

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -80,6 +80,24 @@ build_wheel_linux_python36:
...
+  services:
+    - docker:18.09-dind
+  image: docker:18.09
+  stage: build
+  variables:
+    DOCKER_HOST: tcp://docker:2375/
+    DOCKER_DRIVER: overlay2
+  script:
+    - apk add bash
+    - bash scripts/make_wheel.sh --skip_test --skip_cpp_test --build_number=$CI_PIPELINE_ID --num_procs=4 --docker-python3.7 --skip_doc
```
Comment (TobyRoseman) [inline .gitlab-ci.yml]: Do we want to add the `-e` option, like in #2625?

Comment (abhishekpratapa) [reply .gitlab-ci.yml]: Good call

**Comment by TobyRoseman** [comment]
@abhishekpratapa - is there a reason you didn't also update the `.travis.yml`?

**Comment by TobyRoseman** [comment]
> @abhishekpratapa - is there a reason you didn't also update the `.travis.yml`?

Looks like that's done in #2643

**Pull request merged**